src/ffi/internals/index.ts

```diff
@@ -0,0 +1,57 @@
+// https://github.com/node-ffi/node-ffi/wiki/Node-FFI-Tutorial
+import ffi = require('ffi-napi');
+import path = require('path');
+import { FfiBinding } from './types';
+
+// This is a lookup between process.platform and
+// the platform names used in pact-reference
+const PLATFORM_LOOKUP = {
+  linux: 'linux',
+  darwin: 'osx',
+  win32: 'windows', // yes, 'win32' is what process.platform returns on windows 64 bit
```

Comment:
😆

Reply:
It's "correct", because the name of the windows API layer is "win32". Which is a terrible name 😂